src/graphql/resolvers/queries/event.js

```diff
@@ -51,10 +51,18 @@ export const fieldResolvers = {
           })),
         );
     },
-    sessions: ({ id }, __, { dataSources: { firestore } }) => {
+    sessions: (
+      { id },
+      { onOrAfter, daysAfter },
```

Comment:
litte concerned about the defaults. since these are optional params on sessions. I don't see where we're guarding against things.. example. got onOrAfter but no days. I could have missed it but if we got onOrAfter and not daysAfter we set the default on daysAfter to just 30 days.

Reply:
The logic is in dataSorce/session based on values received

missing both, no filtering done
Has `onOrAfter` but doesn't have `daysAfter`: filters on date and everything after
has `onOrAfter AND daysAfter: filters on date and number of days following as defined with daysAfter
doesn't have onOrAfter` and had `daysAfter`: no filtering is done as the daysAfter value is not used.

I am not testing for valid date as the `Date` scalar we built does that.